lib/livebook_web/live/session_live/app_docker_component.ex

```diff
@@ -80,13 +89,42 @@ defmodule LivebookWeb.SessionLive.AppDockerComponent do
         You can deploy this app in the cloud using Docker. To do that, configure
         the deployment and then use the generated Dockerfile.
       </p>
-      <p class="text-gray-700">
-        <.label>Hub</.label>
-        <span>
-          <span class="text-lg"><%= @hub.hub_emoji %></span>
-          <span><%= @hub.hub_name %></span>
-        </span>
-      </p>
+      <div class="flex gap-12">
+        <p class="text-gray-700">
+          <.label>Hub</.label>
+          <span>
+            <span class="text-lg"><%= @hub.hub_emoji %></span>
+            <span><%= @hub.hub_name %></span>
+          </span>
+        </p>
+        <%= if @hub_type == "team" do %>
+          <%= if @deployment_groups != [] do %>
+            <.form
+              :let={f}
+              for={@deployment_group_form}
+              phx-change="select_deployment_group"
+              phx-target={@myself}
+              id="select_deployment_group_form"
+            >
+              <.select_field
+                help={
+                  ~S'''
+                  Deployment group.
+                  '''
+                }
+                field={f[:id]}
+                options={deployment_group_options(@deployment_groups)}
+                label="Deployment Group"
+              />
+            </.form>
+          <% else %>
+            <p class="text-gray-700">
+              <.label>Deployment Group</.label>
+              <span>No deployment groups available</span>
```

Comment:
Should we have a link to the hub, so it explains more about them? Or maybe we show the same help as above?

Reply:
I think a help is enough for now. But which one are you talking about?